templates/examples/qml/main.qml.tpl

```diff
@@ -0,0 +1,39 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.2
+import QtQuick.Controls 2.15
+{{- range .System.Modules }}
+{{- $module := . }}
+{{- $version := .Version }}
+import {{snake $module.Name }} {{$version.Major}}.{{$version.Minor}}
+{{- end }}
+
+ApplicationWindow {
+    id: appWindow
+    visible: true
+    width: 300
+    height: 300
+    ColumnLayout {
+        spacing: 10
+        id: mainLayout
+        anchors.fill: parent
+
+{{- range.System.Modules }}

```

Comment:
move this to the end of the application window. This leaves the visual elements together.